dsl/api.go

```diff
@@ -150,6 +150,30 @@ func License(fn func()) {
 	eval.IncompatibleDSL()
 }
 
+// Randomizer sets the API example randomizer.
+//
+// Randomizer must appear in a API expression.
+//
+// Randomizer takes a single argument which is an implementation of
+// expr.Randomizer.
+//
+// The default randomizer uses the API name as the seed, to get consistent
+// random examples.
+//
+// Example:
+//
+//    var _ = API("divider", func() {
+//        Random(expr.NewRandom("different seed"))
```

Comment:
The example doesn't seem quite right... Also, would be nice to showcase the deterministic randomizer since that's probably the main use case for `Randomizer`.

Reply:
I'm not 100% sure on the magic comment format for godoc, but I've got two examples here now, I *hope*!